src/App.js

```diff
@@ -45,8 +46,11 @@ class App extends Component {
       this.setState({
         authenticated: true
       }, () => console.log("authenticated.."));
+
+      //console.log(authResult);
+      initUsersConfig(authResult, this.LoadUser.bind(this));
       // following load might have to happen only after udate to sheet
-      load(this.onLoad.bind(this), this.state.uge, new Date().getFullYear());
+      loadMonth(this.onLoad.bind(this), this.state.uge, new Date().getFullYear());
```

Comment:
Vi binder onLoad() 4 gange. Skal vi ikke bare gøre det i constructoren? E.g.
this.onLoad = this.onLoad.bind(this)